bot/commands/say.js

```diff
@@ -47,6 +47,6 @@ Be sure to use the first line and not leave any whitespace/newline immediately a
 
 // TODO: add args
 module.exports.help = {
-	args: '',
+	args: '<channel name> <message>',
```

Comment:
Similar to `<user>`, let's just go with `<channel>` here since multiple formats will be accepted for all commands once we split channels out into a parsing helper.

```suggestion
	args: '<channel> <message>',
```